sdk/eventhub/azure-eventhub/stress/scripts/azure_eventhub_consumer_stress_async.py

```diff
@@ -99,7 +99,15 @@ def parse_starting_position(args):
 debug_level = getattr(logging, args.debug_level.upper(), logging.ERROR)
 
 
-LOGGER = get_logger(args.log_filename, "stress_receive_async", level=debug_level, print_console=args.print_console)
+log_filename = args.log_filename if args.log_filename else "consumer_async.log"
```

Comment:
would it be good to check transport and tack on a "ws" if its webservice ? Can help us when we compare things?

Reply:
so when the file is created in storage, it's created in the folder corresponding to the test name. For ws, it would have ws in the folder name, so I don't think it would be necessary to have it in the file name. thoughts?

Reply:
I figured it would be good once its downloaded, since these changes are being made for easier downloads.

Reply:
oob: adding it in for convenience